spec/support/vcr_cassettes/va_profile/v2/contact_information/delete_address_success.yml

```diff
@@ -5,7 +5,7 @@ http_interactions:
     uri: <VETS360_URL>/contact-information-hub/contact-information/v2/2.16.840.1.113883.4.349/123498767V234859%5ENI%5E200M%5EUSVHA/addresses
     body:
       encoding: UTF-8
-      string: '{"bio":{"addressId":138225,"addressLine1":"4041 Victoria Way","addressLine2":null,"addressLine3":null,"addressPOU":"CORRESPONDENCE","addressType":"Domestic","cityName":"Lexington","countryCodeISO2":null,"countryCodeISO3":"USA","countryName":null,"county":{"countyCode":"21067","countyName":"Fayette County"},"intPostalCode":null,"provinceName":null,"stateCode":"KY","zipCode5":"40515","zipCode4":"4655","originatingSourceSystem":"VETSGOV","sourceSystemUser":"123498767V234859","sourceDate":"2020-02-13T20:47:45.000Z","effectiveStartDate":"2020-02-13T20:47:45.000Z","effectiveEndDate":"2020-02-13T20:47:45.000Z","badAddress":false}}'
+      string: '{"bio":{"addressId":138225,"addressLine1":"4041 Victoria Way","addressLine2":null,"addressLine3":null,"addressPOU":"CORRESPONDENCE","addressType":"Domestic","cityName":"Lexington","country":{"countryName":null,"countryCodeFIPS":null,"countryCodeISO2":null,"countryCodeISO3":"USA"},"county":{"countyCode":"21067","countyName":"Fayette County"},"province":{"provinceName":null,"provinceCode":null},"state":{"stateName":null,"stateCode":"KY"},"intPostalCode":null,"zipCode5":"40515","zipCode4":"4655","originatingSourceSystem":"VETSGOV","sourceSystemUser":"123498767V234859","sourceDate":"2020-02-13T20:47:45.000Z","effectiveStartDate":"2020-02-13T20:47:45.000Z","effectiveEndDate":"2020-02-13T20:47:45.000Z","badAddress":false}}'
```

Comment:
Were these cassettes re-recorded? I saw they original cassetted were a few months old–did the config change since then?

Reply:
This PR was just changed. I did re-record them but didn't want to change the transaction ids all over again.